hist/hist/src/TFormula.cxx

```diff
@@ -1670,55 +1672,57 @@ void TFormula::HandleExponentiation(TString &formula)
    }
 }
 
+namespace {
+
+// Borrowed from RooHelpers.h, should this go to a central place?
+std::vector<std::string> tokenise(const std::string &str, const std::string &delims, bool returnEmptyToken) {
```

Comment:
We generally use US spelling, e.g. `TString::Tokenize`

Reply:
Okay, this should be changed then when the function is moved to `core/foundation`.